src/components/form/field_password/index.d.ts

```diff
@@ -0,0 +1,22 @@
+/// <reference path="../../common.d.ts" />
+
+import { SFC, InputHTMLAttributes } from 'react';
+
+declare module '@elastic/eui' {
+
+  /**
+   * password field type defs
+   *
+   * @see './field_password.js'
+   */
+  export interface EuiFieldPasswordProps {
+    isInvalid?: boolean;
+    inputRef?: (ref: HTMLInputElement) => void;
```

Comment:
Prefer using `React.Ref` type which supports all of the ways to specific a ref, e.g. string, function, or React.createRef

```suggestion
    inputRef?: Ref<HTMLInputElement>;
```

Reply:
Ah, that makes sense. Thanks!